lib/price-import.coffee

```diff
@@ -83,8 +83,8 @@ class PriceImport extends ProductImport
 
   _resolvePriceReferences: (price) =>
     Promise.all [
-      @_resolveReference(@client.customerGroups, 'customerGroup', price.customerGroup, "name=\"#{price.customerGroup?.id}\"")
-      @_resolveReference(@client.channels, 'channel', price.channel, "key=\"#{price.channel?.id}\"")
+      @_resolveReference(@client.customerGroups, 'customerGroup', price.customerGroup, "name=\"#{price.customerGroup?.groupName}\"")
+      @_resolveReference(@client.channels, 'channel', price.channel, "key=\"#{price.channel?.key}\"")
```

Comment:
Second link is not dead - you just do not have permissions (ask Ops). Once you have access you will see that for each importer as reference we use ID and importer knows how to map it (or it might be configurable). In CTP API keys were not always there and still are not everywhere available thats why we wanted to keep ID abstract as by configuration you can define if ID represents (uuid, key, groupName, name, externalId, ...). For external systems it is also easy to export data as they do not need to map their codes/ids to our internal identifiers and it makes import schema consistent when it comes to references for all resources (product, inventories, categories...). You might accept here both abstract id (which can be optionally mapped and has some default mapping) as also native resource specific identifiers like here groupName and key. Take a look at schema and let me know what you think. Important is that if you change it than it has to be consistent with other importers and you should be able to validate input data against schema.